src/main/resources/db/changelog/db.changelog-5.0.xml

```diff
@@ -4,12 +4,91 @@
+        </sql>
+    </changeSet>
+
+    <changeSet id="retrospectively update rivers" author="Defra">
+        <sql>
+            <![CDATA[
+                UPDATE rcr_river set name = 'Allen (South West)', last_modified = now(), version = now() WHERE name = 'Allen (SW)';
+                UPDATE rcr_river set name = 'Avon Water (Hants)', last_modified = now(), version = now() WHERE name = 'Avon (Water Hants)';
+                UPDATE rcr_river set name = 'Blyth (North East)', last_modified = now(), version = now() WHERE name = 'Blyth (NE)';
+                UPDATE rcr_river set name = 'Coron Lake (Anglesea)', last_modified = now(), version = now() WHERE name = 'Coron Lake Anglesea';
+                UPDATE rcr_river set name = 'Llynfi (Ogmore)', last_modified = now(), version = now() WHERE name = 'Llynfi';
+                UPDATE rcr_river set name = 'Rawthey', last_modified = now(), version = now() WHERE name = 'Rawthay';
+                UPDATE rcr_river set name = 'Stour (South West)', last_modified = now(), version = now() WHERE name = 'Stour (SW)';
+                UPDATE rcr_river set name = 'Wygyr (Anglesea)', last_modified = now(), version = now() WHERE name = 'Wygyr Anglesea';
+                UPDATE rcr_river set name = 'Wyre (North West)', last_modified = now(), version = now() WHERE name = 'Wyre NW';
+                UPDATE rcr_river set name = 'Wyre (Wales)', last_modified = now(), version = now() WHERE name = 'Wyre Wales';
+
+                UPDATE rcr_river set name = 'Unknown (Anglian)', last_modified = now(), version = now() WHERE name = 'Unknown Anglian';
+                UPDATE rcr_river set name = 'Unknown (Eng and Wales)', last_modified = now(), version = now() WHERE name = 'Unknown Eng and Wales';
+                UPDATE rcr_river set name = 'Unknown (Midlands)', last_modified = now(), version = now() WHERE name = 'Unknown Midland';
+                UPDATE rcr_river set name = 'Unknown (North East)', last_modified = now(), version = now() WHERE name = 'Unknown NE';
+                UPDATE rcr_river set name = 'Unknown (North West)', last_modified = now(), version = now() WHERE name = 'Unknown NW';
+                UPDATE rcr_river set name = 'Unknown (Southern)', last_modified = now(), version = now() WHERE name = 'Unknown Southern';
+                UPDATE rcr_river set name = 'Unknown (South West)', last_modified = now(), version = now() WHERE name = 'Unknown SW';
+                UPDATE rcr_river set name = 'Unknown (Thames)', last_modified = now(), version = now() WHERE name = 'Unknown Thames';
+                UPDATE rcr_river set name = 'Unknown (Wales)', last_modified = now(), version = now() WHERE name = 'Unknown Wales';
+            ]]>
+        </sql>
+    </changeSet>
+
+    <changeSet id="retrospectively update region" author="Defra">
+        <sql>
+            <![CDATA[
+                UPDATE rcr_catchment set region_id = 9, last_modified = now(), version = now() where name = 'Unknown Eng and Wales';
```

Comment:
I'm a bit confused why we're switching adding the Goyt to changelog 6 and then all of this to changelog 5. Why aren't we leaving changelog 5 as it is and putting all of this into changelog 6?

Reply:
Hi Phil, I haven't put the updates in preprod yet. So I thought it would make sense to do the migrations in the order they should've occurred. So the retrospective adding first, then the river goyt.